lib/APIDigester/ModuleAnalyzerNodes.cpp

```diff
@@ -2215,8 +2218,81 @@ static parseJsonEmit(SDKContext &Ctx, StringRef FileName) {
   }
   return {std::move(FileBufOrErr.get()), Result};
 }
+enum class ConstKind: uint8_t {
+  String = 0,
+  Int,
+};
+
+struct ConstExprInfo {
+  StringRef filePath;
+  ConstKind kind;
+  unsigned offset = 0;
```

Comment:
As far as I understand, `offset` here is measured in number of characters from the start of the buffer, and `length` is the difference between token `loc`s of the Expr where we found this value. 

I'm not sure these are useful values to have here, what's the thinking behind including them?